js/scripts.js

```diff
@@ -77,7 +80,12 @@ async function fetchDataDocuments() {
             option.selected = true;
         }
 
-        options.year.prepend(option);
+        elements.options.year.prepend(option);
+    }
+
+    for (let yearStats of tabletop.sheets('stats').all()) {
+        stats.set(yearStats['Year'], yearStats);
+        delete yearStats['Year'];
```

Comment:
Still not clear on why this is necessary.

Reply:
It's not really necessary, just makes sure the `yearStats` object only includes actual stats (states, countries, etc.) and not the year of those stats, since the year is already the key of the `Map` the stats are in.